extensions/amp-date-picker/0.1/amp-date-picker.css

```diff
@@ -17,15 +17,31 @@
 @import '../../../third_party/react-dates/_datepicker.css';
 
 
-amp-date-picker,
-.SingleDatePicker_picker,
-.DateRangePicker_picker {
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif;
+amp-date-picker.i-amphtml-element {
+  display: block;
+  position: relative;
 }
 
-amp-date-picker * {
+amp-date-picker[mode="overlay"] .i-amphtml-date-picker-container {
+  position: absolute;
+  top: calc(100% + 10px);
+  left: 10px;
+  z-index: 10;
+}
+
+amp-date-picker[fullscreen] .i-amphtml-date-picker-container,
+amp-date-picker[with-full-screen-portal],
+amp-date-picker[with-full-screen-portal] .i-amphtml-date-picker-container {
+  position: absolute;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+}
+
+amp-date-picker .i-amphtml-date-picker-container {
   box-sizing: border-box;
-  font-family: inherit;
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif;
```

Comment:
This now is not `public` anymore but I think we should allow them to change the font size. Can this be at `amp-date-picker` level or moved to a public CSS class ?